cli-plugins/apps/how-to-guides/workload-from-private-git.hbs.md

```diff
@@ -0,0 +1,8 @@
+# Create a workload from a private Git repository
```

Comment:
Is this topic needed?
I has no content only links to other content
Is there a way to build this into an existing topic?